pkg/client/client.go

```diff
@@ -187,7 +187,11 @@ func (c *Client) CreateOrUpdateSecurityContextConstraints(s *secv1.SecurityConte
 		return errors.Wrap(err, "retrieving SecurityContextConstraints object failed")
 	}
 
-	_, err = sccclient.Update(s)
+	// the CRD version of SCC appears to require this.  We can try to chase why later.
+	required := s.DeepCopy()
+	required.ResourceVersion = existing.ResourceVersion
```

Comment:
you can see this done in this file for other resources like https://github.com/openshift/cluster-monitoring-operator/pull/343/files#diff-fb87476fa66849c8514ba700e715b4b6R270